ja/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -0,0 +1,285 @@
+
+<br>
+
+**33. Regularization ― The regularization procedure aims at avoiding the model to overfit the data and thus deals with high variance issues. The following table sums up the different types of commonly used regularization techniques:**
+
+&#10230;　正則化 - 正則化はモデルが過学習するのを避ける目的としており，したがってバリアンスが大きくなる問題に対処します．次の表は一般的に使用されるいくつかの正則化法をまとめたものです．
+
+<br>
+
+**34. [Shrinks coefficients to 0, Good for variable selection, Makes coefficients smaller, Tradeoff between variable selection and small coefficients]**
+
+&#10230; [係数を0にする,変数選択に適する,係数を小さくする,変数選択と係数を小さくすることのトレードオフ]
+
+<br>
+
+**35. Diagnostics**
+
+&#10230; 分析
+
+<br>
+
+**36. Bias ― The bias of a model is the difference between the expected prediction and the correct model that we try to predict for given data points.**
+
+&#10230; バイアス - モデルのバイアスとは，予測するあるデータ点における，予測した結果の期待値と正しいモデルによる結果との差です．
+
+<br>
+
+**37. Variance ― The variance of a model is the variability of the model prediction for given data points.**
+
+&#10230; バリアンス - モデルのバリアンスとは，予測するあるデータ点における，予測した結果の分散です．
+
+<br>
+
+**38. Bias/variance tradeoff ― The simpler the model, the higher the bias, and the more complex the model, the higher the variance.**
+
+&#10230; バイアス・バリアンストレードオフ - よりシンプルなモデルではバイアスが高くなり，より複雑なモデルはバリアンスが高くなります．
+
+<br>
+
+**39. [Symptoms, Regression illustration, classification illustration, deep learning illustration, possible remedies]**
+
+&#10230; [症状,回帰モデルでの図,分類モデルでの図,深層学習での図,可能な解決策]
+
+<br>
+
+**40. [High training error, Training error close to test error, High bias, Training error slightly lower than test error, Very low training error, Training error much lower than test error, High variance]**
+
+&#10230; [高い訓練誤差,訓練誤差がテスト誤差に近い，高いバイアス,訓練誤差がテスト誤差より少しだけ小さい,極端に小さい訓練誤差,訓練誤差がテスト誤差に比べて非常に小さい,高いバリアンス]
+
+<br>
+
+**41. [Complexify model, Add more features, Train longer, Perform regularization, Get more data]**
+
+&#10230; [より複雑なモデルを試す,特徴量を増やす，より長く学習する,正則化を導入する,データ数を増やす]
+
+<br>
+
+**42. Error analysis ― Error analysis is analyzing the root cause of the difference in performance between the current and the perfect models.**
+
+&#10230; エラー分析 - エラー分析は完璧なモデルと現在のモデル間の性能差の主要な要因を分析することです．
+
+<br>
+
+**43. Ablative analysis ― Ablative analysis is analyzing the root cause of the difference in performance between the current and the baseline models.**
+
+&#10230; アブレーション分析 - アブレーション分析は，ベースラインとするモデルと現在のモデル間の性能差の主要な要因を分析することです．
+
+<br>
+
+**44. Regression metrics**
+
+&#10230; 回帰評価指標
+
+<br>
+
+**45. [Classification metrics, confusion matrix, accuracy, precision, recall, F1 score, ROC]**
+
+&#10230; [分類評価指標,混同行列,正解率,適合率,再現率,F値,ROC]
+
+<br>
+
+**46. [Regression metrics, R squared, Mallow's CP, AIC, BIC]**
+
+&#10230; [回帰評価指標,R二乗,マローズのCp,AIC,BIC]
+
+<br>
+
+**47. [Model selection, cross-validation, regularization]**
+
+&#10230; [モデル選択，交差検証，正則化]
+
+<br>
+
+**48. [Diagnostics, Bias/variance tradeoff, error/ablative analysis]**
+
+&#10230; [分析，バイアス・バリアンストレードオフ，エラー・アブレーション分析]
```

Comment:
```suggestion
&#10230; [解析（分析），バイアス・バリアンストレードオフ，エラー・アブレーション分析]
```

Reply:
I think `診断` or `診断方法` is better. We need discuss this topic.
https://github.com/shervinea/cheatsheet-translation/pull/99/files#r349865518

Reply:
Yep, I think it might be ok with "診断", however when Google searching with combination words "機械学習" and "診断", I look at many headlines related to the medical topics in Japan. The combination words "機械学習" and ("解析" or "分析") pick up headlines related to the machine learning in Japan. Though my approaching to find best translated word not be good one, how do you think about that ?

Reply:
Yes, you're right. "診断" is a bit weird. This section is about trouble shooting when your model doesn't work as expected. But "分析" sounds like "問題分析". How about "問題分析" ?

Reply:
Thank you for your nice comment. I agree with "問題分析". It will be definitely fit to original nuance in English.

Reply:
Thank you for your nice comment. I agree with "問題分析". It will be definitely fit to original nuance in English.

Reply:
As far as I know, 打診 in Japanese had a similar word sense derivation like diagnostics in English: from medical usages to common ones.